domain-models-runtime/src/main/java/org/folio/rest/persist/Criteria/Criteria.java

```diff
@@ -431,60 +242,46 @@ public Criteria setOperation(String operation) {
     return this;
   }
 
-  /** prefix the criteria with an alias - for example,
-   * if the query does something like FROM table1 t1
-   * then we need to prefix the jsonb for example like
-   * t1.jsonb */
-  public Criteria setAlias(String alias){
+  /**
+   * prefix the criteria with an alias - for example, if the query does
+   * something like FROM table1 t1 then we need to prefix the jsonb for example
+   * like t1.jsonb
+   */
+  public Criteria setAlias(String alias) {
     this.alias = alias;
     return this;
   }
 
-  public String getAlias(){
+  public String getAlias() {
     return this.alias;
   }
 
-  public boolean isJSONB() {
-    return isJSONB;
-  }
-
   /**
-   * set this to false if not a jsonb field criteria
-   * For example: the _id is not in the jsonb object hence would be false if criteria based on _id
+   * set this to false if not a jsonb field criteria For example: the _id is not
+   * in the jsonb object hence would be false if criteria based on _id
    */
   public Criteria setJSONB(boolean isJSONB) {
     this.isJSONB = isJSONB;
     return this;
   }
 
-  public boolean isNotQuery() {
-    return isNotQuery;
-  }
-
```

Comment:
As long as there is the setter setNotQuery we can have the getter isNotQuery. Possible usage: Criterion.java should use the getter instread of directly reading the variable.

Reply:
ok